pipeline/scripts/electron-build-id.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/*
+This number identifies an internally-maintained Electron build
+used in our releases. Each Electron update in package.json requires
+a corresponding change here.
+*/
+exports.electronBuildId = '7912306';
```

Comment:
I thought about running this only in PR builds, but I'm not sure. On one hand it'll simplify electron prototyping locally, but on the other it'll defer issues until PR-time.